src/Connection/Replication/SentinelReplication.php

```diff
@@ -538,7 +538,15 @@ protected function assertConnectionRole(NodeConnectionInterface $connection, $ro
     {
         $role = strtolower($role);
         $actualRole = $connection->executeCommand(RawCommand::create('ROLE'));
-
+        if ($actualRole instanceof ErrorResponseInterface) {
+            $info = $connection->executeCommand(RawCommand::create('INFO REPLICATION'));
```

Comment:
`RawCommand::create()` expects a list of arguments, so it should be `RawCommand::create('INFO', 'REPLICATION')`.